lib/post_creator.rb

```diff
@@ -420,7 +420,7 @@ def create_embedded_topic
 
   def update_uploads_secure_status
     return if !SiteSetting.secure_media?
-    @post.update_uploads_secure_status
+    @post.update_uploads_secure_status(source: "post creator")
```

Comment:
This could be a one-liner

```ruby
def update_uploads_secure_status!
  @post.update_uploads_secure_status(source: "post creator") if SiteSetting.secure_media?
end
```